chapters/shaders.adoc

```diff
@@ -2498,6 +2498,86 @@ In other shader stages, each invocation in a subgroup must: be in the same
 
 Only <<limits-subgroup-supportedStages, shader stages that support subgroup
 operations>> have defined subgroups.
+
+[NOTE]
+.Note
+====
+In shaders, there are two kinds of uniformity that are of primary interest to
+applications: subgroup uniform and dynamically uniform.
+
+While dynamically uniform appears to imply subgroup uniform,
+it is not necessarily the case for shader stages without defined workgroups.
+
+For shader stages with defined workgroups, this assumption holds.
```

Comment:
nit: "this assumption" -> I understand what this refers to, but there's strictly speaking no "assumption" stated above. (My suggestion above would make the assumption explicit.)

Reply:
Reworded.